languageFiles/es.json

```diff
@@ -9,7 +9,7 @@
   "CC Options": "Opciones de subtitulado",
   "On": "Sí",
   "Off": "No",
-  "Discover": "Descubrir",
+  "Discovery": "Descubrir",
```

Comment:
Modified to Discovery because that's how it's named in the english language file. Could you please check that ?